API/NoDocs/Microsoft.Extensions.Http.AutoClient/AutoClientException.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Microsoft.Extensions.Http.AutoClient'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Http.AutoClient;
+
+public class AutoClientException : Exception
+{
+    public AutoClientHttpError? HttpError { get; }
+    public string Path { get; }
+    public int? StatusCode { get; }
+    public AutoClientException(string? message, string path, AutoClientHttpError? error = null);
```

Comment:
There is no overload with innerExeption. That's a standard. Should it be added here?

Reply:
Added